qt/applications/workbench/workbench/plugins/test/test_workspacewidget.py

```diff
@@ -66,6 +66,21 @@ def test_detector_table_shows_with_workspace(self):
             self.ws_widget._do_show_detectors([self.ws_names[0]])
         self.assert_widget_type_exists(MATRIXWORKSPACE_DISPLAY_TYPE)
 
+    @mock.patch('workbench.plugins.workspacewidget.plot', autospec=True)
+    def test_plot_with_plot_bin(self,mock_plot):
+        self.ws_widget._do_plot_bin([self.ws_names[0]], False, False)
+        mock_plot.assert_called_once_with(unittest.mock.ANY,errors=False, overplot=False, wksp_indices=[0],
```

Comment:
Just noticed that `unittest.mock` needs to be just `mock` as `mock` is a submodule of `unittest` and not an attribute on it (this is why macOS is failing). Not sure why this works on Windows.